webapp/app/routes.py

```diff
@@ -317,6 +316,18 @@ def error_500(error):
             'An uncaught error occurred', exc_info=error.original_exception)
         return render_template('error/500.html', header_title=_('500.header-title'), js_needed=False), 500
 
+    @app.errorhandler(EricaRequestTimeoutError)
+    def error_500(error):
+        current_app.logger.error(
+            'An Erica Request Timeout error occurred', exc_info=error.original_exception)
+        return render_template('error/500.html', header_title=_('500.header-title'), js_needed=False), 500
```

Comment:
Imo we should use `error/erica.html ` template to indicate that it is not an external server error but a problem in the connection with Erica. Otherwise, the same error would be shown to the user as before.

Reply:
Yes, this was my thought too. Maybe we could also place a kind of error code in the template (error map) if a user is asking for support (future task)?

Reply:
Yes, great idea. Maybe you could add a ticket for that in Jira?